bude_hezky/request/request_maker.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from requests.exceptions import HTTPError
+
+SUCCESSFUL_STATUS_CODE_THRESHOLD = 299;
+
+class RequestException(Exception):
+    pass
+
+def make_request(url, params, response=None):
+    try:
+        response = response or requests.get(url, params=params)
+        response.raise_for_status()
+    except HTTPError as http_err:
+        raise RequestException(f'HTTP error occurred: {http_err}') from http_err
+    except Exception as e:
+        raise RequestException(f'Other error occurred: {e}') from e
+    else:
+        if response.status_code > SUCCESSFUL_STATUS_CODE_THRESHOLD:
```

Comment:
to samý uděláš přes

```python
response.raise_for_status()
```

Reply:
Ajo, to tam máš už. Takže tohle je zbytečný asi?

Reply:
A jo 😆

Reply:
Jsem špatně pochopil, co to dělá :D